src/io/platform/f4xx/I2Cf4xx.cpp

```diff
@@ -0,0 +1,264 @@
+#include <EVT/io/platform/f4xx/I2Cf4xx.hpp>
+
+#include <EVT/io/pin.hpp>
+#include <EVT/io/platform/f4xx/GPIOf4xx.hpp>
+
+//#include <EVT/utils/log.hpp>
```

Comment:
Why did you comment out all the logging stuff in this file?

Reply:
Looks like it was not being used in any of the F4 functions but is being used in f3. So I uncommented it